src/bucket.ts

```diff
@@ -636,8 +636,8 @@ class Bucket extends ServiceObject {
   }
   signer?: URLSigner;
 
-  private instanceRetryValue?: boolean;
-  private instancePreconditionOpts?: PreconditionOptions;
+  instanceRetryValue?: boolean;
+  instancePreconditionOpts?: PreconditionOptions;
```

Comment:
Any reason we want these `public`? I usually default to avoid having class properties public to avoid breaking changes in the future (say we want to change the type or variable name).

If we want them public, but we don't want anyone to modify, we could use `readonly` here.

Reply:
changed instanceRetryValue back to private. instancePreconditionOpts needs to be public so we can modify it for the tests. @ddelgrosso1 are you okay with this? just want to make sure we very intentionally make this decision.

Reply:
I am ok with this. Think I removed it to try out something and never put it back.